webpack/InsightsCloudSync/InsightsCloudSync.js

```diff
@@ -25,13 +26,17 @@ const InsightsCloudSync = ({ settingsUrl, syncInsights }) => {
               target="_blank"
               rel="noopener noreferrer"
             >
-              access.redhat.com <Icon name="external-link" size="xs" />
+              access.redhat.com <Icon name="external-link" />
             </a>
             <br />
             {__("2. Copy the token to 'Red Hat Cloud token' setting: ")}
-            <a href={settingsUrl} target="_blank" rel="noopener noreferrer">
+            <a
+              href={foremanUrl('/settings?search=name+%3D+rh_cloud_token')}
```

Comment:
Are there any helpers for url construction either in foreman or in the framework? It feels a bit prone to errors to construct urls manually.

Reply:
thanks, I assume we could use `urijs` for that :+1: